### PR TITLE
[react-jsonschema-form] fix missing `label` property

### LINKED ISSUE #33582: react-jsonschema-form type declaration does not compile
- [x] I tried using the `@types/react-jsonschema-form` package and had problems.
- [x] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [x] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).
- [x] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @iamdanfox @iplus26 @phbou72 @LucianBuzzo @sbusch 

The index.d.ts file does not compile on React 16.6.3 & typescript 3.3.3333
Issue: React.ReactElement required 1 type argument but none is provided.

### PR DESCRIPTION
**Pull request by medfreeman**
- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [X] Provide a URL to documentation or source code which provides context for the suggested changes: 
  - `React.ReactElement` #33582 
  - Missing `label` property on `WidgetProps` interface -> https://github.com/mozilla-services/react-jsonschema-form/blob/68ab7a4e4932f35afa7cd646e8dedd5c1d579cb6/src/components/widgets/CheckboxWidget.js#L41
- [X] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/react-jsonschema-form/index.d.ts [hunk 1]
```diff
@@ -87,6 +88,7 @@ declare module "react-jsonschema-form" {
         formContext: any;
         onBlur: (id: string, value: string) => void;
         onFocus: (id: string, value: string) => void;
+        label: string;
```
Comment (sbusch) [inline types/react-jsonschema-form/index.d.ts]: Had to search for this prop but found it finally on CheckboxWidget. So :+1 from me for this change

**Review by sbusch — COMMENTED** [review]
Are those `React.ReactElement<React.ReactNode>` changes really required?

According to the [current definition for `ReactElement`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L91), the type parameter defaults to `any` when being omitted.

@RobbeB at #33582 uses React version 16.6.3. We don't know the version of `@types/react`, but this could already be fixed there?!

**Comment by RobbeB** [comment]
> @RobbeB at #33582 uses React version 16.6.3. We don't know the version of `@types/react`, but this could already be fixed there?!

I use `@types/react` version 16.7.17

**Comment by medfreeman** [comment]
@sbusch you're right, my bad.

They've been added [here](https://github.com/DefinitelyTyped/DefinitelyTyped/commit/2fc40ca501f2cc734deb9313fa1032e3381a0e26#diff-96b72df8b13a8a590e4f160cbc51f40cR86), and probably released in [`@types/react`] v16.8.4 according to the date.

I just tested with `@types/react` v16.8.8 and it works without the changes.
I'll just revert the corresponding commit right now.
About the `label`, sorry you had to search, i put the link the PR's text.

**Review by sbusch — APPROVED** [review]

**Comment by sbusch** [comment]
@medfreeman No problem, thanks for your contribution!

**Pull request merged**